.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
I don't think we need this. It would be more appropriate in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) than at the project level.